### PR TITLE
PHP-FPM Extend "access.format" configuration

### PR DESCRIPTION
**Pull request by RoSk0**

### CODE REVIEW

File: install/fpm/configuration.xml [hunk 1]
```diff
@@ -918,6 +918,10 @@
           </tbody>
          </tgroup>
         </table>
+        <para>
+         For additional <parameter>access.format</parameter> configuration parameters see
+         https://github.com/php/php-src/blob/HEAD/sapi/fpm/www.conf.in .
+        </para>
```
Comment (Girgias) [inline install/fpm/configuration.xml:924]: These should be documented in the manual, and not redirect people the sources IMHO.

In any case this needs proper markup for linking and entity usage to refer to the git repo.

**Review by Girgias — CHANGES_REQUESTED** [review]

File: install/fpm/configuration.xml (hunk 1 above)
Comment (RoSk0) [reply install/fpm/configuration.xml]: Replaced the link with the actual descriptions

File: install/fpm/configuration.xml [hunk 2]
```diff
@@ -859,31 +876,43 @@
...
             </entry>
-            <entry>query string</entry>
+            <entry>the query string</entry>
            </row>
            <row>
             <entry>
              <literal>%Q</literal>
             </entry>
-            <entry>the glue between %q and %r</entry>
+            <entry>the '?' character if query string exists</entry>
```
Comment (Girgias) [inline install/fpm/configuration.xml:909]:
```suggestion
            <entry>the <literal>?</literal> character if query string exists</entry>
```

File: install/fpm/configuration.xml [hunk 3]
```diff
@@ -859,31 +876,43 @@
...
            </row>
            <row>
             <entry>
              <literal>%r</literal>
             </entry>
-            <entry>request URI</entry>
+            <entry>the request URI (without the query string, see %q and %Q)</entry>
```
Comment (Girgias) [inline install/fpm/configuration.xml:915]:
```suggestion
            <entry>the request URI (without the query string, see <literal>%q</literal> and <literal>%Q</literal>)</entry>
```

File: install/fpm/configuration.xml [hunk 4]
```diff
@@ -895,25 +924,41 @@
...
             <entry>
              <literal>%t</literal>
             </entry>
-            <entry>time</entry>
+            <entry>
+             server time the request was received. It can accept a 
+             <literal>strftime(3)</literal> format:
+             <literal>%d/%b/%Y:%H:%M:%S %z</literal> (default).
+             The <literal>strftime(3)</literal> format must be encapsulated in a
+             <literal>%{<strftime_format>}t</literal> tag, e.g. for a ISO8601
```
Comment (Girgias) [inline install/fpm/configuration.xml:952]: This is not valid XML as it is creating a tag, use the relevant XML entities of `<` and `>`

**Review by Girgias — COMMENTED** [review]
Some markup comments, but I would like @bukka to review the content.

File: install/fpm/configuration.xml (hunk 4 above)
Comment (RoSk0) [reply install/fpm/configuration.xml]: Fixed

**Comment by Girgias** [comment]
This now conflicts, can you rebase please?

**Comment by RoSk0** [comment]
https://github.com/php/doc-en/pull/3772 managed to get in before this.

Left only what was missing in https://github.com/php/doc-en/pull/3772 .

**Review by bukka — APPROVED** [review]
LGTM

**Pull request merged**